tests/operator/main.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/kyma-project/docker-registry/components/operator/api/v1alpha1"
+	"github.com/kyma-project/docker-registry/tests/operator/dockerregistry"
+	"github.com/kyma-project/docker-registry/tests/operator/logger"
+	"github.com/kyma-project/docker-registry/tests/operator/namespace"
+	"github.com/kyma-project/docker-registry/tests/operator/utils"
+)
+
+var (
+	testTimeout = time.Minute * 10
+)
+
+func main() {
+	ctx, cancel := context.WithTimeout(context.Background(), testTimeout)
+	defer cancel()
+
+	log, err := logger.New()
+	if err != nil {
+		fmt.Printf("%s: %s\n", "unable to setup logger", err)
+		os.Exit(1)
+	}
+
+	log.Info("Configuring test essentials")
+	client, err := utils.GetKuberentesClient()
+	if err != nil {
+		log.Error(err)
+		os.Exit(1)
+	}
+
+	log.Info("Start scenario")
+	err = runScenario(&utils.TestUtils{
+		Namespace: fmt.Sprintf("dockerregistry-test-%s", uuid.New().String()),
+		Ctx:       ctx,
+		Client:    client,
+		Logger:    log,
+
+		Name:                     "default-test",
+		DockerregistryDeployName: "internal-docker-registry",
+		RegistryName:             "dockerregistry-docker-registry",
+		UpdateSpec: v1alpha1.DockerRegistrySpec{
+			HealthzLivenessTimeout: "20",
```

Comment:
Is this field used anywhere? If not, I would suggest to remove it or replace it with something like `// TODO: ...`